server/flaskr/utils/__init__.py

```
@@ -0,0 +1,2 @@
+from .response import *
```

Ruff (0.15.15)

[error] `from .response import *` used; unable to detect undefined names

(F403)